end_app/weather_app/data_services.py

```diff
@@ -0,0 +1,22 @@
+from .get_weather import get_weather, get_forecast, get_air_pollution
+
+
+class DataView:
```

Comment:
To nie jest widok, a serwis, WeatherService byłoby lepszą nazwą, bo samo `data` też nie za dużo mówi